### PR TITLE
add support for the `ROLLBACK` statement

### PR DESCRIPTION
**Pull request by kovrus**
## Summary of the changes / Why this improves CrateDB
This improves the support for clients that are based on the Postgres wire protocol.

## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: sql-parser/src/main/java/io/crate/sql/tree/RollbackStatement.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "ROLLBACK";
+    }
+
+    @Override
+    public <R, C> R accept(AstVisitor<R, C> visitor, C context) {
+        return visitor.visitRollback(this, context);
```
Comment (seut) [inline sql-parser/src/main/java/io/crate/sql/tree/RollbackStatement.java]: We could let them all calling a e.g. `visitIgnoredStatement` method to simplify the code (doesn't really make sense to implement custom visitors which all do nothing).

Comment (kovrus) [reply sql-parser/src/main/java/io/crate/sql/tree/RollbackStatement.java]: if it is ok with you, i'll do it and sql syntax doc in the separate PR(s).

**Comment by kovrus** [comment]
Currently, it is not required and not completely correct semantically.

**Comment by r2dedios** [comment]
Please reconsider implementing this feature.

**Pull request closed without merging**